ws/handlers.go

```diff
@@ -200,3 +200,36 @@ func MeStopTypingHandler(event Event, c *Client) error {
 
 	return nil
 }
+
+func UpdateMembershipHandler(event Event, c *Client) error {
+	var updateMembershipEvent UpdateMembership
+
+	if err := json.Unmarshal(event.Payload, &updateMembershipEvent); err != nil {
+		return fmt.Errorf("bad payload in update membership :%v", err)
+	}
+
+	var updateMembership UpdateMembership
+
+	updateMembership.UserID = updateMembershipEvent.UserID
+	updateMembership.OrganizationID = updateMembershipEvent.OrganizationID
+	updateMembership.Role = updateMembershipEvent.Role
+
+	data, err := json.Marshal(updateMembership)
+
```

Comment:
you don't need to unmarshal and then Marshal it, you can just send it directly as the payload is exact same :)